apps/hash-frontend/src/components/sandbox/framed-block/framed-block.tsx

```diff
@@ -27,13 +24,16 @@ export const FramedBlock: FunctionComponent = () => {
 
   const initialData = properties ? JSON.parse(properties) : undefined;
 
-  const [blockProperties, setBlockProperties] = useState<
-    BlockProtocolEntity | undefined
-  >(initialData);
+  const [blockProperties, setBlockProperties] = useState<Entity | undefined>(
```

Comment:
Should we rename this to `blockEntity`? As it's `Entity` not `EntityPropertyObject`

Reply:
I _almost_ deleted these files becasue this sandbox isn't used, but it started to unpick some cross-frame considerations that probably are worth keeping in place for when we re-introduce it. I wouldn't spend any effort on updating the logic though beyond getting it to pass linting.